README.md

```diff
@@ -2,20 +2,17 @@
+  
+Highways are an important part of our infrastructure and are used by a lot of people every day for different reasons: To commute to work, transport goods and to go on holidays. Thus a general speed limit may have significant consequences for many people. In particular, we want to analyze the effects of a general speed limit on the throughput of a two-lane highway.  
+While many (European) countries have a general speed limit in place, Germany is one of few countries that doesn't have such a limit and where controversial debates surrounding a general speed limit happen regularly [5, 6]. Both sides seemingly use the same argument: A general speed limit decreases travel time by reducing traffic congestion, while without such a speed limit travel time is decreased because people can drive faster and thus reach their destination quicker. We want to see which of the argumentations hold. 
 
 ## The Model
-We model a straight multi-lane highway, where cars enter from the left and exit to the right. To determine the acceleration of each car at a given time step, we use a modified version of the Intelligent Driver Model (IDM) [1]. Since we are modeling a multi-lane highway, we also need to model lane-changing behaviour. This is achieved by utilizing the lane-changing model MOBIL [2].  
-To analyse the results of our simulation, we define our independent variables as time, if a general speed limit is in place and the proportions of different drivers/cars, and our dependent variables as the location of cars and their associated velocity/acceleration at a given time step. The dependent variables can easily be measured by storing the corresponding parameters at every time step in a file.  
 
-(Define dependent and independent variables you want to study. Say how you want to measure them.) (Why is your model a good abtraction of the problem you want to study?) (Are you capturing all the relevant aspects of the problem?)
+We model a straight two-lane highway, where cars enter from the left and exit to the right. To determine the acceleration of each car at a given time step, we use the Intelligent Driver Model (IDM) [1, 3]. Since we are modeling a two-lane highway, we also need to model lane-changing behaviour. This is achieved by utilizing the lane-changing model MOBIL [2, 4].  
+To analyse the results of our simulation, we define our independent variables as time, if a general speed limit is in place and the proportions of different types of vehicles, and our dependent variables as the location of cars and their associated velocity/acceleration at a given time step. The dependent variables can easily be measured by storing the corresponding parameters at every time step in a file.  
```

Comment:
Does this sentence make sense? I don't get it

Reply:
I don't know, I am just answering the questions in parentheses.